### PR TITLE
Docs/update readme

### PR DESCRIPTION
**Pull request by thomasheartman**
# Description

This PR makes some minor changes to the readme in this repo:

- Add a language annotation to the `pip install` command
- Lowercase the `Python` language annotation for code blocks
- Merge/reword contents to match what is (currently) on docs.getunleash.io

## Background

We're making a change to the documentation on docs.getunleash.io. Keeping the docs in sync with the SDKs and their readmes is difficult and a lot of manual work, so we've decided to move to generating the SDK documentation from the project readmes. Moving forward, all SDK-specific documentation will be replaced with the SDKs' respective github readmes. (Refer to https://github.com/Unleash/unleash/pull/2809 for more background.)

This PR aims to update this project readme to make sure that no information is lost in this content.

## Type of change

Please delete options that are not relevant.
- [x] This change **is** a documentation update

# How Has This Been Tested?

N/A

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

---

Please let me know if you have any questions or thoughts ☺️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,35 +43,42 @@ client = UnleashClient(
...
     return True
 
 client.is_enabled("my_toggle", fallback_function=custom_fallback)
 ```
 
-You can also use the `fallback_function` argument to replace the obsolete `default_value` by using a lambda that ignores its inputs:
+You can also use the `fallback_function` argument to replace the obsolete `default_value` keyword argument by using a lambda that ignores its inputs. Whatever the lambda returns will be used as the default value.
 
-```Python
+```python title="Use fallback_function to provide a default value"
```
Comment (thomasheartman) [inline README.md:75]:
```suggestion
```python
```

**Comment by rishiraj88** [comment]
Thanks, @thomasheartman

**Comment by rishiraj88** [comment]
Thanks, @coveralls

**Review by sighphyre — APPROVED** [review]
LGTM

**Pull request merged**